evap/staff/tests/test_views.py

```diff
@@ -1896,6 +1906,46 @@ def test_remove_participants_proportional_reward_points(self):
         )
         self.assertNotIn("The removal as participant has granted the user &quot;d@institution.example.com&quot;", page)
 
+    def test_questionnaire_with_answers_warning(self):
+        page = self.app.get(self.url, user=self.manager)
+        self.assertIn(
```

Comment:
Probably a good case for [`self.assertInHtml`](https://docs.djangoproject.com/en/4.0/topics/testing/tools/#django.test.SimpleTestCase.assertInHTML)